components/pkg-export-kubernetes/src/manifest.rs

```diff
@@ -105,3 +105,62 @@ impl Manifest {
         Ok(())
     }
 }
+
+#[cfg(test)]
+mod tests {
+    use super::*;
+
+    #[test]
+    fn test_manifest_generation() {
+        let mut m = Manifest {
+            metadata_name: "hi-f84iq".to_owned(),
+            habitat_name: "hi".to_owned(),
+            image: "core/hi".to_owned(),
+            count: 3,
+            service_topology: Default::default(),
+            service_group: Some("group1".to_owned()),
+            config_secret_name: Some("alphabravocharlie".to_owned()),
```

Comment:
Too much flying. ;)

Reply:
haha, @krnowak unfortunately not even enough flying. :( @LiliC i didn't quite get the emoji combo :)